wallet/src/multisig_account.rs

```diff
@@ -157,23 +162,21 @@ impl MultiSigAccount {
             Vec::with_capacity(raw_aggregated_signature.len() + raw_aggregated_commitment.len());
         combined.extend_from_slice(&aggregated_commitment.to_bytes());
         combined.extend_from_slice(aggregated_signature.as_bytes());
```

Comment:
@sisou  A small oversight from my initial implementation that you can pick up in this PR. `raw_aggregated_signature` and `raw_aggregated_commitment` can be used to extend the `combined` slice. The byte representation is now computed twice.